src/TailwindBuilder.php

```diff
@@ -61,7 +61,11 @@ public function runBuild(
             throw new \InvalidArgumentException('The --poll option is not supported in Tailwind CSS v4.0.0 and later.');
         }
 
-        $arguments = ['-c', $this->configPath, '-i', $inputPath, '-o', $this->getInternalOutputCssPath($inputPath)];
+        $arguments = ['-i', $inputPath, '-o', $this->getInternalOutputCssPath($inputPath)];
+
+        if (!$binary->isV4()) {
```

Comment:
In v4 of Tailwind CSS the `tailwind.config.js` is indeed optional because it has sensible defaults now, but you still can use it if you need to override some defaults. So, IMO, completely excluding it if we're on v4 is not quite correct. Could you confirm @kbond ?

Probably we need a different check here to decide if we should include the config or no

Reply:
Hm, if the `tailwind.config.js` still can be used in v4 and the option `-c` was not removed from it yet, probably we should just check if the `$this->configPath` file exists, and if it does - include it with `-c`. If not - do not add that `-c` config? Does it make sense to you guys? //cc @seb-jean as the reporter of this issue.

Reply:
OK, thanks for confirming 👍🏼